Comanche055/P20-P25.agc

```diff
@@ -782,7 +782,7 @@ PI/4.0		2DEC	.785398164
 #	MARKDATA, BASE ADDRESS OF MARK DATA
 #	REFSMMAT, ROTATION MATRIX FROM STABLE MEMBER TO BASIC REF. COORD. SYSTEM
 #
-# SUROUTINES CALLED:
```

Comment:
Approved, but colon should be replaced with hyphen (`SUBROUTINES CALLED-`) ([source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0578.jpg))